core/src/main/java/org/apache/gravitino/hook/CatalogHookDispatcher.java

```diff
@@ -71,6 +72,10 @@ public Catalog createCatalog(
       String comment,
       Map<String, String> properties)
       throws NoSuchMetalakeException, CatalogAlreadyExistsException {
+    // Check whether the current user exists or not
+    AuthorizationUtils.checkCurrentUser(
```

Comment:
Do we only need to check `createXXX` operation?

Reply:
We should check every operations actually. 
We don't support meta authentication now. I think this should be included to meta authentication. 
But for some managers, we don't a dispatcher interface for them like ownerManager and TagManager.  We will add the interfaces first. I'm not sure to do them now. So I just make a minor improvement to check `createXXX` operation first.